### PR TITLE
Fix confusing wording on verifying smart contracts page

### PR DESCRIPTION
**Pull request by OdairTrujillo**
Changing "former" by "formal"

## Description
Just a mistyping error

## Related Issue
Need to change "former" word by "formal" word in the context of "formal verification"

### CODE REVIEW

**Review by minimalsm — COMMENTED** [review]
Hey @OdairTrujillo, this isn't a typo.

'The former' refers to the first item in a list of two items. [More on this here, if you aren't familiar.](https://www.merriam-webster.com/words-at-play/former-and-latter-usage#:~:text=Former%20refers%20to%20something%20that,in%20a%20group%20of%20several.). 

But perhaps we should tighten up the language here... if it isn't clear to everyone.

**Comment by OdairTrujillo** [comment]
@minimalsm Hi, sorry for this misinterpretation. You are right, maybe you sould be more specific in this text.

File: src/content/developers/docs/smart-contracts/verifying/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar: true
...
 
-It is important to make the distinction between "source code verification" and "[formal verification](/developers/docs/smart-contracts/formal-verification/)". The former, which will be explained in detail below, refers to verifying that the given source code of a smart contract in a high-level language (e.g. Solidity) compiles to the same bytecode to be executed at the contract address. The latter describes verifying the correctness of a smart contract, meaning the contract behaves as expected. Although context-dependent, contract verification usually refers to source code verification.
+It is important to make the distinction between "source code verification" and "[formal verification](/developers/docs/smart-contracts/formal-verification/)". The formal, which will be explained in detail below, refers to verifying that the given source code of a smart contract in a high-level language (e.g. Solidity) compiles to the same bytecode to be executed at the contract address. The latter describes verifying the correctness of a smart contract, meaning the contract behaves as expected. Although context-dependent, contract verification usually refers to source code verification.
```
Comment (corwintines) [inline src/content/developers/docs/smart-contracts/verifying/index.md:10]:
```suggestion
It is important to make the distinction between "source code verification" and "[formal verification](/developers/docs/smart-contracts/formal-verification/)". Source code verification, which will be explained in detail below, refers to verifying that the given source code of a smart contract in a high-level language (e.g. Solidity) compiles to the same bytecode to be executed at the contract address. Formal verification describes verifying the correctness of a smart contract, meaning the contract behaves as expected. Although context-dependent, contract verification usually refers to source code verification.
```

We could tighten it up with something like this

Comment (corwintines) [reply src/content/developers/docs/smart-contracts/verifying/index.md:10]: Let me know what you think @OdairTrujillo. Feel free to commit this suggestion if you agree and we can pull it in.

**Review by minimalsm — APPROVED** [review]
Pulled in your changes @corwintines

Thanks for spotting and raising this @OdairTrujillo 

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @OdairTrujillo for docs, ideas

File: src/content/developers/docs/smart-contracts/verifying/index.md (hunk 1 above)
Comment (OdairTrujillo) [reply src/content/developers/docs/smart-contracts/verifying/index.md]: This feels better, but maybe we can add an initial sentence to the final paragraph, something like "_On the other hand, formal verification desbribes verifying ...._"

**Pull request merged**